src/xunit.performance.api/ETWProfiler.cs

```diff
@@ -107,9 +119,196 @@ public static void Profile(string assemblyFileName, string sessionName, Action a
 
             TraceEventSession.MergeInPlace(userFullFileName, Console.Out);
             WriteInfoLine($"ETW Tracing Session saved to \"{userFullFileName}\"");
+            collectOutputFilesCallback(userFullFileName);
+
+            var assemblyModel = GetAssemblyModel(assemblyFileName, userFullFileName, sessionName, performanceTestMessages);
+            var xmlFileName = Path.Combine(outputDirectory, $"{Path.GetFileNameWithoutExtension(userFullFileName)}.xml");
+            WriteXmlFile(xmlFileName, assemblyModel);
+            collectOutputFilesCallback(xmlFileName);
+
+            var mdFileName = Path.Combine(outputDirectory, $"{Path.GetFileNameWithoutExtension(userFullFileName)}.md");
+            DataTable dt = CreateStatistics(assemblyModel);
+            var mdTable = MarkdownHelper.GenerateMarkdownTable(dt);
+            MarkdownHelper.Write(mdFileName, mdTable);
+
+            collectOutputFilesCallback(mdFileName);
+            Console.WriteLine(mdTable);
+        }
+
+        private static DataTable CreateStatistics(AssemblyModel assemblyModel)
+        {
+            var dt = new DataTable();
```

Comment:
What happens if we add an extra column, or forget to add a column, compared to what MarkdownHelper is expecting? Is there any way to typecheck the columns we add and their respective data types?